src/screens/buy/debit.tsx

```diff
@@ -13,13 +15,15 @@ import { useSignedMoonPayUrl } from './useSignedUrl';
 export const Debit = () => {
   const styles = useBuyStyles();
   const url = useSignedMoonPayUrl();
+  const theme = useThemeSelector();
+  const MoonPaythemeIcon = theme === ThemesEnum.light ? IconNameEnum.MoonPayBlack : IconNameEnum.MoonPayWhite;
 
   return (
     <>
       <Divider size={formatSize(16)} />
       <WhiteContainer>
         <View style={styles.providerLogo}>
-          <Icon name={IconNameEnum.MoonPay} size={formatSize(160)} />
+          <Icon name={MoonPaythemeIcon} size={formatSize(160)} />
```

Comment:
instead of changing the whole SVG, you could set `color` prop with a value from `useColors()`.  
you will need to change the original svg, as `svgr.config.js` change `const orange = '#FF5B00'` color to the prop value